web-client/src/presenter/state.ts

```diff
@@ -870,3 +871,10 @@ export type CreateCaseIrsForm = {
   irsNoticeFileUrl?: string;
   cityAndStateIssuingOffice?: string;
 };
+
+export type ViewerDocument = {
+  docketEntryId: string;
+  documentTitle?: string; // Should this be required?
```

Comment:
I don't believe we can guarantee that there will be a documentTitle here, so I think this is correct